### PR TITLE
Refactor to make checks more DRY

### PR DESCRIPTION
**Pull request by mkunkel**

### CODE REVIEW

File: lib/bobo/exception_helper.rb [hunk 1]
```diff
@@ -7,5 +7,6 @@ def ignore_exceptions
         # do nothing
       end
     end
+    module_function :ignore_exceptions
```
Comment (edubkendo) [inline lib/bobo/exception_helper.rb]: if you are including this elsewhere, you likely do not need `module_function` here. It is for allowing you to call a function directly on a module like `Bobo::ExceptionHelper.ignore_exceptions`. But I might be missing something.

**Comment by edubkendo** [comment]
One comment/question, otherwise:

thumbsup1

File: lib/bobo/exception_helper.rb (hunk 1 above)
Comment (mkunkel) [inline lib/bobo/exception_helper.rb]: Really, that is only in there so that I can test it. I supposed I could create a class to inherit in the test.

Comment (edubkendo) [inline lib/bobo/exception_helper.rb]: Yeah, that's what I'd do I think. No big deal but it makes the code that little bit less intention revealing. I'm not sure, did you try using `send(:ignore_exceptions)` without the module_function? That might also work.

Comment (mkunkel) [inline lib/bobo/exception_helper.rb]: I just tried that and it doesn't work. I'm pushing the change with a test class in a moment.

Comment (robuye) [inline lib/bobo/exception_helper.rb]: depends how it's going to be tested, but you coud include it in an object directly.

```
o = Object.new
o.singleton_class.include(Bobo::ExceptionHelper)
expect(o.module_function).to eq(1)
```

Comment (edubkendo) [inline lib/bobo/exception_helper.rb]: @robuye with the awesome! 
amazed

**Comment by mkunkel** [comment]
Nice! Thanks @robuye

**Pull request merged**